### ISSUE TITLE
[LegacyBuff] Only auto-attacks are affected

### ISSUE THREAD

**Issue by BraynStorm** [HellscreamWoW/Tracker#2493]
VIdeo shows (when its done uploading, approx 15mins): https://youtu.be/iRCrOBN51qw

**Comment by Cyuty** [comment]
Since is on legacybuff i will post here to. Sinister Strike and revealing strike work's doing alot of dmg but, eviscerate and other don't.

**Comment by BraynStorm** [comment]
Oh so it's some attack only? Huh...

**Comment by Cyuty** [comment]
idk but it shoud be for all skill but some are left outside

**Comment by BraynStorm** [comment]
Yeah I know it should buff all skills but your comment suggest that it only affects some.
